lib/rtcpeerconnection/chrome.js

```diff
@@ -248,10 +279,6 @@ util.delegateMethods(
 // for the unreliable MediaStreamTrack#addtrack event. Do this only if
 // the native RTCPeerConnection has not implemented 'ontrack'.
 function maybeDispatchTrackEvents(peerConnection, mediaStreamTracks) {
-  if ('ontrack' in PeerConnection.prototype) {
-    return;
-  }
-
```

Comment:
Why are we removing this?

Reply:
@markandrus Now that `ontrack` exists in `PeerConnection.prototype`, we should dispatch `track` events for this case as well. So we're removing this check.